marklogic-data-hub/src/main/java/com/marklogic/hub/impl/VersionInfo.java

```diff
@@ -16,18 +16,21 @@
 
     private String hubVersion;
     private String markLogicVersion;
+    private String clusterName;
 
     public static VersionInfo newVersionInfo(HubClient hubClient) {
```

Comment:
I expect to rename this to SystemInfo in a future JIRA ticket as well, didn't want to increase the blast radius of this PR though. Also note that "clusterName" is appropriate here because this is a DHF class, and it doesn't know specifically about DHS, which is where the concept of "serviceName" exists.